packages/babel-plugin-transform-runtime/src/index.js

```diff
@@ -17,12 +43,38 @@ export default declare((api, options, dirname) => {
   const {
     corejs,
     helpers: useRuntimeHelpers = true,
-    regenerator: useRuntimeRegenerator = true,
+    regenerator = true,
     useESModules = false,
     version: runtimeVersion = "7.0.0-beta.0",
     absoluteRuntime = false,
+    targets: optionsTargets = {},
+    configPath = process.cwd(),
```

Comment:
While I am aware that introducing `configPath` and `ignoreBrowserslistConfig` is to align the interface of `preset-env`, I prefer to hold off a bit as we are considering changing the browserslists related options in Babel 8. https://github.com/nicolo-ribaudo/babel-rfcs/blob/top-level-targets/rfcs/0000-top-level-browserslist.md

Instead of introducing multiple `browserslists` options in 7.10 and then deprecate them in Babel 8, we can add `targets` only and iterate later.

Reply:
Done.

Reply:
Unresolving because I realized I assumed you'd want `ignoreBrowserslistConfig` set to `true` in `getTargets`. Do you think that's the right move?

My logic is that if we set `ignoreBrowserslistConfig` to `false`, then users have no way to have their browserslist config ignored other than removing it, which could mess with any other tooling they use. If we set `ignoreBrowserslistConfig` to `true`, then it's a little inconvenient for users because they may have to specify their targets in two configs, but at least we haven't put them in an impossible situation.

Reply:
👍That looks reasonable to me.